src/d/actor/d_a_bomb2.cpp

```diff
@@ -618,17 +618,17 @@ namespace daBomb2 {
     }
 
     void Act_c::set_nut_exp_interval() {
-        if(field_0x738 > attr().field_0xA) {
-            field_0x738 = attr().field_0xA;
+        if(mBombTimer > attr().field_0xA) { // attr().field_0xA appears to be the bomb timer cap
```

Comment:
Since `attr()` just inlines `L_attr`, this comment hinting at a potential purpose belongs there.
Example
```c++
            /* 0x0A */ s16 field_0xA; // Appears to be the bomb timer cap
```